lib/bloc/contact_interface_bloc.dart

```diff
@@ -0,0 +1,6 @@
+import '../model/contact.dart';
+import 'bloc.dart';
+
+abstract class ContactInterfaceBloc extends Bloc {
```

Comment:
We usually name interfaces like i_contact_bloc as name of file and IContactBloc as name of interface

Reply:
Create a new folder call impl or implementation, inside that you can move all the files relative to files that implement interfaces and outside impl add all the interfaces files

Reply:
got it